content/docs/tasks/security/mutual-tls/index.md

```diff
@@ -69,11 +69,19 @@ Please check [Istio identity](/docs/concepts/security/#istio-identity) for more
 
 ## Verify mutual TLS configuration
 
-You can use the `istioctl` tool to check the effective mutual TLS settings. To identify the authentication policy and destination rules used for the
-`httpbin.default.svc.cluster.local` configuration and the mode employed, use the following command:
+For this task, you may want to get the pod ID of the client app (`sleep`) with the following command:
 
 {{< text bash >}}
-$ istioctl authn tls-check httpbin.default.svc.cluster.local
+$ SLEEP_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+{{< /text >}}
+
+You can use the `istioctl` tool to check the effective mutual TLS settings. The command requires the client pod as the destination rule depends on the client namespace.
+You can also provide the destination service to filter the status to that service only.
+
+For example, to identify the authentication policy for service `httpbin.default.svc.cluster.local` and destination rules to it, as seen from pod `${SLEEP_POD}`, use the following command:
```

Comment:
Be consistent with the rest of the docs: 
```suggestion
To identify the authentication policy for the `httpbin.default.svc.cluster.local` service and its destination rules as seen from the `${SLEEP_POD}` pod, use the following command:
```

Reply:
Done